ql/src/java/org/apache/hadoop/hive/ql/ddl/table/drop/DropTableAnalyzer.java

```diff
@@ -51,7 +55,14 @@ public void analyzeInternal(ASTNode root) throws SemanticException {
     Table table = getTable(tableName, throwException);
     if (table != null) {
       inputs.add(new ReadEntity(table));
-      outputs.add(new WriteEntity(table, WriteEntity.WriteType.DDL_EXCLUSIVE));
+
+      boolean tableWithSuffix = (HiveConf.getBoolVar(conf, ConfVars.HIVE_ACID_CREATE_TABLE_USE_SUFFIX)
+          || HiveConf.getBoolVar(conf, ConfVars.HIVE_ACID_LOCKLESS_READS_ENABLED))
+        && AcidUtils.isTransactionalTable(table)
+        && table.getSd().getLocation().matches(PATH_WITH_VERSION_ID);
+
+      outputs.add(new WriteEntity(table, 
```

Comment:
It would be good to see this in the log. Minimally with debug level

Reply:
it's already logged by DbTxnManager when calling AcidUtils.makeLockComponents